pkg/handler/employee/employee_test.go

```diff
@@ -98,7 +98,7 @@ func TestHandler_GetProfile(t *testing.T) {
 			w := httptest.NewRecorder()
 
 			ctx, _ := gin.CreateTestContext(w)
-			ctx.Request = httptest.NewRequest("POST", fmt.Sprintf("%s", "/api/v1/employee/2655832e-f009-4b73-a535-64c3a22e558f/employee-status"), nil)
+			ctx.Request = httptest.NewRequest("POST", fmt.Sprintf("%s", "/api/v1/employee/profile"), nil)
```

Comment:
attribute should be plural (`employees`)

Reply:
dạ ok anh